tools/perf/components/browser_type.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
```

Comment:
nit: should all those files be python3 or vpython3?

Reply:
According to upstream style, all `#!/usr/bin/env` in secondary (that can't be run itself) .py files should be removed.
Fixed now.